Functions.Templates/Templates/DurableFunctionsEntityHttpStart-TypeScript/index.ts

```diff
@@ -0,0 +1,19 @@
+﻿﻿import { AzureFunction, Context, HttpRequest } from "@azure/functions"
+import * as df from "durable-functions"
+
+const httpStart: AzureFunction = async function (context: Context, req: HttpRequest) {
+    const client = df.getClient(context);
+    const id: string = context.bindingData.id;
```

Comment:
```suggestion
    const id: string = req.params.id;
```

... to match how we access the route param in [this template](https://github.com/Azure/azure-functions-templates/blob/dev/Functions.Templates/Templates/DurableFunctionsHttpStart-JavaScript/index.js#L5)